eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2597,17 +2598,78 @@ public function testMoveSubtreeHidden()
         // Load new parent location
         $newParentLocation = $locationService->loadLocation($demoDesignLocationId);
 
+        // Move location from "Home" to "Demo Design" (not container)
+        $locationService->moveSubtree($locationToMove, $newParentLocation);
+    }
+
+    /**
+     * Test for the moveSubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::moveSubtree()
```

Comment:
Please use `@covers` instead for all new or changed test cases, without method call parentheses, like that:
```suggestion
     * @covers \eZ\Publish\API\Repository\LocationService::moveSubtree
```